wire-ios/WireCommonComponents/FontScheme.swift

```diff
@@ -50,6 +51,11 @@ public enum FontWeight: String, CaseIterable {
     case black
 }
 
+public extension Font {
+    static var sfMonoSmall = Font.custom("SF Mono", size: 12)
```

Comment:
Could you please confirm with the design team that we will be using new fonts?

Reply:
Yes. A new font was added to make the fingerprint more readable.